CookBookProject/IOSCookBook/IOSCookBook/Scenes/Main/MainViewController.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  MainViewController.swift
+
+import UIKit
+import Foundation
+import MGArchitecture
+
+class MainViewController: UIViewController, BindableType {
+    @IBOutlet private weak var tableView: UITableView!
+    
+    var viewModel: MainViewModel!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+    }
+    
+    func bindViewModel() {
+        let input = MainViewModel.Input(
+            loadTrigger: Driver.just(()),
+            selectTrigger: tableView.rx.itemSelected.asDriver()
+        )
+        let output = viewModel.transform(input)
+    }
```

Comment:
không handle output của viewModel à ?

Reply:
tôi để đến phần sau tải lên table view luôn, tui in thử & nó có giá trị rồi